core/plugin_manager.js

```diff
@@ -79,16 +79,29 @@ Blockly.PluginManager.prototype.getPlugin = function(id) {
  * @template T
  */
 Blockly.PluginManager.prototype.getPlugins = function(type, sorted) {
-  var plugins = [];
   var typeKey = String(type).toLowerCase();
   var pluginIds = this.typeToPluginId_[typeKey];
```

Comment:
nit: Could we make `typeToPluginId_` be `typeToPluginId**s**`? The 's' implies returning multiply ids instead of a single id.

Reply:
sgtm